.coderabbit.yml

```
@@ -11,5 +11,78 @@ reviews:
+            newlines-between: always
+
+        # Error prevention
+        no-unused-vars: error
+        no-undef: error
+        no-console: error
+        no-debugger: error
+        no-alert: error
+
+        # Code style
+        no-trailing-spaces: error
+        no-multiple-empty-lines: [error, { max: 1 }]
```

yamllint (1.35.1)

[error] too many spaces inside braces

(braces)